### PR TITLE
Added a document with background/howtos on condition mapping table

### LINKED ISSUE #12149: Document the process of generating the observation mapping CSV from the RCTC
### User Story
As a reportstream dev I want the process for generating the observation mapping csv documented
so that I can generate new versions as needed.

### Description/Use Case
Document the process for turning the RCTC 

### Risks/Impacts/Considerations
- The original ticket (#11948) mentions the csv also contains mappings for AOE questions from LOINC.org.

### Dev Notes
- See #11948 for an example csv and links to the RCTC

### Acceptance Criteria
- Markdown document added for rctc <-> observation-mapping.csv process

### PR DESCRIPTION
**Pull request by thetaurean**
This PR adds a markdown file with background & procedures on the condition mapping table:
- How to fetch the table to a local csv
- *TBD* How to update the table via CLI using public sources
  - pending outcome of #11510
- How to update the table via local csv
- How to create a local csv from public sources

It also includes a listing of public sources used.

Test Steps:
1. N/A

## Changes
- Added `observation-mapping-table.md` to `docs/getting-started/standard-operating-procedures`

### Testing
N/A

## Linked Issues
- Fixes #12149 

## Todo
- Instructions for using #11510 (pending) to update the table

### CODE REVIEW

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (mkalish) [inline prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:5]:
```suggestion
The observation mapping table maps a plethora of conditions, observations, and aoe questions to codes in various code
```

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (mkalish) [inline prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:13]: Is there a ticket for this we can reference here?

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (mkalish) [inline prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:38]: 😄 Can we elaborate on what peril means here?  Is this just saying be careful creating a CSV be hand

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (mkalish) [inline prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:25]: Just want to confirm that if this table already existed this is an update and doesn't drop what's there?

**Review by mkalish — APPROVED** [review]
LGTM, a few questions I'd like to work through though before merging.

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (thetaurean) [reply prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:25]: Correct 👍 `create` handles that use case i.e. there is no separate `update` command.

```
Usage: prime lookuptables [OPTIONS] COMMAND [ARGS]...

  Manage lookup tables

Options:
  -h, --help  Show this message and exit

Commands:
  list             List the lookup tables
  get              Fetch the contents of a lookup table
  create           Create a new version of a lookup table
  activate         Activate a specific version of a lookup table
  diff             Generate a difference between two versions of a lookup
                   table
  loadall          Load all the tables stored as CSV in the specified
                   directory
  compare-mapping  Compares a sender compendium against an observation mapping
                   lookup table, outputting an annotated CSV
```

```
Usage: prime lookuptables create [OPTIONS]

  Create a new version of a lookup table

Options:
  -e, --env <name>       Connect to <name> environment. Choose between
                         [local|test|staging|prod]
  -i, --input-file PATH  Input CSV file with the table data
  -s, --silent           Do not generate diff or ask for confirmation
  -a, --activate         Activate the table upon creation
  -n, --name TEXT        The name of the table to perform the operation on
  --show-table           Always show the table to be created
  -f, --force            Force the creation of new table(s) even if it is
                         already exist
  -h, --help             Show this message and exit
```

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (thetaurean) [reply prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:38]: Main peril I was consider was using the columns listed in this markdown doc and failing to check the CSV.
(i.e. CSV gets updated, has new columns, markdown update is forgotten).

**Comment by thetaurean** [comment]
I believe all feedback is addressed. Will merge this evening barring any further feedback.

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (arnejduranovic) [inline prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:75]: Do we need more elaboration in this section? It's not clear exactly how to do this, or is it self-explanatory?

**Review by arnejduranovic — CHANGES_REQUESTED** [review]

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (thetaurean) [reply prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:56]: `LOINC.org` in the latest xlsx ✅ Good call on double-checking this 👍

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (thetaurean) [reply prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:25]: It was my understanding that the AOE questions will need to be manually updated. Not that they change frequently.

I've added a note to the effect of all of the above 👍 .

File: prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md
Comment (thetaurean) [reply prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:75]: They are sufficiently self-explanatory I believe.

**Comment by thetaurean** [comment]
@arnejduranovic I addressed your feedback. Also made it clearer on the TBD section that it's waiting for results of #11510.

I'll wait for your 👍 to merge.

**Review by arnejduranovic — APPROVED** [review]

**Pull request merged**
